lightning/src/util/config.rs

```diff
@@ -206,7 +206,14 @@ pub struct ChannelConfig {
 	/// This cannot be changed after a channel has been initialized.
 	///
 	/// Default value: true.
-	pub commit_upfront_shutdown_pubkey: bool
+	pub commit_upfront_shutdown_pubkey: bool,
+	/// Limit on the maximum value of dusted offered HTLCs to the counterparty at any given time
+	/// to limit holder balance exposure to HTLCs. Dusted HTLCs are defined as any HTLC lower
+	/// than counterparty's `dust_limit_satoshis` sumed up with their expexted onchain claim
+	/// cost at minimal mempools congestion.
+	///
+	/// Default value: 25_350_000 (330 sat + `min_feerate_per_kw` * HTLC_SUCCESS_TX_WEIGHT / 1000) * 50 * 1000
```

Comment:
I don't think there's any reason to keep the current default here. Maybe 10k sats for max exposure of $5? Maybe 5k if we want to go a little lower?

Reply:
Picked up 5k as we might expect this threshold to become more sensible with time for the average node operator.